pkg/component/component.go

```diff
@@ -237,7 +237,8 @@ func PushLocal(client *occlient.Client, componentName string, applicationName st
 		scanner := bufio.NewScanner(pipeReader)
 		for scanner.Scan() {
 			line := scanner.Text()
-			yellowFprintln(out, line)
+			// switching to color.output as color.output passed to this function and cast to io.writer is not working on windows. Will solve in future.
```

Comment:
You need to add `TODO:` to this so it's searchable in the future.

Have 1 line describing what's happening and another line with `TODO` saying that we will fix this in the future. 

You also need to capitalize `switching`

Reply:
So something like this is acceptable:

```go
// color.Output is temporarily used as there is a type cast error when passing in color.Output from cmd/create.go
// TODO: Fix this in the future more upstream in the code at cmd/create.go rather than within this function.
```